packages/toolkit/src/auth/ui/vue/authForms/manageBuilderId.vue

```diff
@@ -170,7 +170,9 @@ abstract class BaseBuilderIdState implements AuthForm {
      */
     async getSubmitButtonText(): Promise<string> {
         if (!(await this.anyBuilderIdConnected())) {
-            return 'Use for free with AWS Builder ID'
+            return this.name === 'CodeWhisperer'
+                ? 'Use for free, no AWS Account required'
+                : 'Use for free with AWS Builder ID'
```

Comment:
Whats the motivation for this change?
`'Use for free, no AWS Account required'` is rather misleading right? A builder ID is an account after all...